core/src/main/java/com/rudderstack/android/sdk/core/RudderTraits.java

```diff
@@ -262,7 +262,8 @@ public static String getUsername(Map<String, Object> traitsMap) {
      */
     public RudderTraits() {
         RudderContext rudderContext = RudderElementCache.getCachedContext();
```

Comment:
We don't actually need to look for RudderContext here

Reply:
yes. removed.